server/taskflows/hpccloud/taskflow/nwchem.py

```diff
@@ -0,0 +1,290 @@
+            cluster_id = cluster_id[0].value
+            model = ModelImporter.model('cluster', 'cumulus')
+            cluster = model.load(cluster_id, user=user, level=AccessType.ADMIN)
+            cluster = model.filter(cluster, user, passphrase=False)
+            kwargs['cluster'] = cluster
+
+        profile_id = parse('cluster.profileId').find(kwargs)
+        if profile_id:
+            profile_id = profile_id[0].value
+            model = ModelImporter.model('aws', 'cumulus')
+            profile = model.load(profile_id, user=user, level=AccessType.ADMIN)
+            kwargs['profile'] = profile
+
+        kwargs['image_spec'] = self.NWCHEM_IMAGE
+        kwargs['next'] = setup_input.s()
+
+        super(NwChemTaskFlow, self).start(
+            setup_cluster.s(
+                self, *args, **kwargs))
+
+    def terminate(self):
+        self.run_task(nwchem_terminate.s())
+
+    def delete(self):
+        for job in self.get('meta', {}).get('jobs', []):
+            job_id = job['_id']
+            client = _create_girder_client(
+            self.girder_api_url, self.girder_token)
+            client.delete('jobs/%s' % job_id)
+
+            try:
+                client.get('jobs/%s' % job_id)
+            except HttpError as e:
+                if e.status != 404:
+                    self.logger.error('Unable to delete job: %s' % job_id)
+
+
+# TODO: move to common class
+def _create_girder_client(girder_api_url, girder_token):
+    client = GirderClient(apiUrl=girder_api_url)
+    client.token = girder_token
+
+    return client
+
+@cumulus.taskflow.task
+def nwchem_terminate(task):
+    cluster = task.taskflow['meta']['cluster']
+    client = _create_girder_client(
+                task.taskflow.girder_api_url, task.taskflow.girder_token)
+    terminate_jobs(
+        task, client, cluster, task.taskflow.get('meta', {}).get('jobs', []))
+
+def update_config_file(task, client, *args, **kwargs):
+
+    ini_file_id = kwargs['input']['iniFile']['id']
+
+    _, path = tempfile.mkstemp()
+    fileContents = ''
+
+    task.logger.info('Downloading configuration file.')
+    try:
+        with open(path, 'w') as fp:
+            client.downloadFile(ini_file_id, path)
+
+        with open(path, 'r') as fp:
+            for line in fp.readlines():
+                if 'load' in line:
+                    task.logger.info('Expanding path to geometry files.')
+                    # Find the geometry file and check for input in file path
+                    for w in line.split():
+                        (filePath, fileExt) = os.path.splitext(w)
+                        if fileExt == '.xyz' or fileExt == '.pdb':
+                            (fileDir, fileName) = os.path.split(w)
+                            if (not fileDir.startswith('input')):
+                                newPath = 'input' + os.sep + filePath
+                                line = line.replace(filePath, newPath, 1)
+                fileContents += line
```

Comment:
Ok, so the plan it to copy ( we can just create a symlink ) into the job directory so nwchem can find the geometry file, rather than having to patch the input file.